operator_ui/__tests__/connectors/reducers/jobs.test.js

```diff
@@ -75,12 +76,30 @@ describe('connectors/reducers/jobs', () => {
       data: {
         specs: {
           a: { id: 'a' }
-        },
-        runs: {}
+        }
       }
     }
     const state = reducer(undefined, action)
 
     expect(state.jobs.items).toEqual({ a: { id: 'a' } })
   })
+
+  it('RECEIVE_DELETE_SUCCESS deletes items', () => {
+    const upsertAction = {
+      type: UPSERT_JOB,
+      data: {
+        specs: {
+          a: { id: 'a' }
+        }
+      }
+    }
+    const preDeleteState = reducer(undefined, upsertAction)
+
+    expect(preDeleteState.jobs.items).toEqual({ a: { id: 'a' } })
+    const deleteAction = {
+      type: RECEIVE_DELETE_SUCCESS,
+      data: 'a'
+    }
+    const postDeleteState = reducer(undefined, deleteAction)
```

Comment:
This should include an assertion that `a` is no longer in `items`

Reply:
@rupurt slipped by :P